### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by brmatola**
I saw this jekyll site earlier today and really liked it.  As I was trying to customize it for my uses, I found there were some references throughout the code to data (images etc.) hosted on @ashutosh2k12's personal website.  I changed it to point to the data in the project itself.  
Also, I decluttered the _config.yml file by moving the data in there to separate files in the _data directory.
And finally, the default.html layout seemed a little large and hard to follow, so I pulled the code in it out into a handful of include files with more definite purposes.
Thanks so much for making a great jekyll template!  I hope this is helpful!

### CODE REVIEW

**Comment by GreatEmerald** [comment]
While you're at it, how about adding _site to .gitignore? It makes little sense to keep generated files in the repository, when they will always be different for each template user and cause merge conflicts.

**Comment by brmatola** [comment]
Hadn't ever put something in .gitignore after it had been tracked before.  _site's no longer there, though.
